README.md

```diff
@@ -1,25 +1,34 @@
-## Build the docker image
+## Installation with Docker
 
-`sudo ./build.sh`
+Edit the `.env` with your own settings. A secret key can be generated in Python REPL:
 
-## Run the docker container
+```
+>>> from django.core.management.utils import get_random_secret_key
+>>> get_random_secret_key()
+```
+
+Then launch the dockers with:
 
 `sudo docker-compose up -d`
 
-## Initialize the database
+You certainly want to apply databases migrations with:
 
-`sudo ./makemigrations.sh`
+`sudo docker/migrate.sh`
 
-`sudo ./migrate.sh`
+## Browse the admin
 
-## Run the tests
+Create a super-user:
 
-`sudo ./test.sh`
+```
+sudo docker exec -it mvapi_web_1 python ./manage.py createsuperuser
+```
 
-## Create a super-user
+Visit the admin page at [http://localhost:8012/admin/](http://localhost:8012/admin/).
 
-`sudo docker exec -it mvapi_web_1 python ./manage.py createsuperuser`
```

Comment:
Why do you want to remove this?

Reply:
I just moved it to distinct the installation steps, from the administration stuffs.